app/Filament/Resources/AchievementSetResource/RelationManagers/GameHashesRelationManager.php

```diff
@@ -112,7 +117,8 @@ public function table(Table $table): Table
                             ->title('Success')
                             ->body('Successfully marked hash as incompatible.')
                             ->send();
-                    }),
+                    })
+                    ->visible(fn () => $user->can('markGameHashAsIncompatible', [AchievementSet::class])),
```

Comment:
This should be applied to the "Remove" link too.

Reply:
Updated in latest.